release-notes/v1_8.md

```diff
@@ -212,6 +213,12 @@ Coding assistance for JavaScript embedded in HTML is back! You get code completi
 
 ![JavaScript editing in HTML](images/1_8/javascript-in-html.gif)
 
+### TypeScript 2.1
```

Comment:
The in-product release notes have trouble linking to h3 headers with punctuation. We may just want to remove the highlight link or change the title to something like "TypeScript Update".